src/main/java/ch/uzh/ifi/hase/soprafs21/controller/UsersApiController.java

```diff
@@ -91,4 +96,10 @@ public UsersApiController(NativeWebRequest request, UserService userService, Cha
         }
     }
 
+    @Override
+    public ResponseEntity<Void> updateLocation(@ApiParam(value = "Numeric ID of the user to update",required=true) @PathVariable("userId") Long userId,@ApiParam(value = "Coordinate object that needs to be updated in user with userId" ,required=true )  @Valid @RequestBody CoordinateDto coordinateDto) throws Exception {
+        Point newLocation = new GeometryFactory().createPoint(new Coordinate(coordinateDto.getLongitude(), coordinateDto.getLatitude()));
+        userService.updateUserLocation(userId, newLocation);
+        return ResponseEntity.noContent().build();
+    }
```

Comment:
True. This will be added later(as well as other checks). 
I decided to go for the implementation of a simple functionality first(as it stated in the Milestone requirement). Later on I will add and test other flows.